File: app/src/main/java/com/example/android/indonesiaandroidkejar3/MainActivity.java
package com.example.android.indonesiaandroidkejar3;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v7.app.AppCompatActivity;

/**
 * Created by MYWINDOWS on 05/11/2017.
 */

public class MainActivity extends AppCompatActivity {
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        new AsyncTask().execute();
    }
}

File: Asynctask/app/src/main/java/com/example/android/indonesiaandroidkejar3/MainActivity.kt
package com.example.android.indonesiaandroidkejar3

import android.os.Bundle
import android.support.v7.app.AppCompatActivity

/**
 * Created by MYWINDOWS on 05/11/2017.
 */

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        AsyncTask().execute()
    }
}
